### PR TITLE
Add model components, icons, stylesheet

### PR DESCRIPTION
**Pull request by Yukino2002**
Added the filterbutton, modelcard components, the stylesheet for models page, and the necessary public files.

### CODE REVIEW

File: new-website/deepchem/components/Models/FilterButton.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+/**
+ * @component
+ * @param {Object} props - props passed to the component
+ * @param {Array} props.category - array of all the available filters
+ * @param {String} props.name - name of the particular filter
+ * @param {Image} props.image - image associated to the filter
+ * @return {JSX.Element} - A JSX element representing the FilterButton
+ */
+const FilterButton = ({ category, name, image }) => {
```
Comment (rbharath) [inline new-website/deepchem/components/Models/FilterButton.js:14]: Can you add to the docstring explaining what this component is supposed to do?

**Review by rbharath — COMMENTED** [review]
As a request on top of my minor comment below, can we start tracking a list of all components for the website in the docs? Might be good to add to the README

File: new-website/deepchem/components/Models/FilterButton.js (hunk 1 above)
Comment (Yukino2002) [reply new-website/deepchem/components/Models/FilterButton.js:14]: Added the information in the docstring for both the components.

**Comment by Yukino2002** [comment]
> As a request on top of my minor comment below, can we start tracking a list of all components for the website in the docs? Might be good to add to the README

Sir, as per our earlier discussion, we had talked about generating the html docs for the documentation of the components, and deploying them on readthedocs after the end of the milestones. We had introduced the js docstring for the same reason, so do we still need to mention it on the readme?

**Review by rbharath — APPROVED** [review]
LGTM

**Pull request merged**